local_server/qwen3_tts_server/local_server.py

```
@@ -0,0 +1,348 @@
+                if "text" in data and not msg_type: msg_type = "legacy.text"
+
+                if msg_type == "input_text_buffer.append":
+                    text_fragment = data.get("text", "")
+                    sentence_buffer += text_fragment
+
+                elif msg_type in ("input_text_buffer.commit", "legacy.text"):
+                    if msg_type == "legacy.text":
+                        sentence_buffer += data.get("text", "")
+
+                    # 正则智能断句
+                    parts = re.split(r'([。！？.!?\n]+)', sentence_buffer)
```

Ruff (0.15.0)

[warning] String contains ambiguous `！` (FULLWIDTH EXCLAMATION MARK). Did you mean `!` (EXCLAMATION MARK)?

(RUF001)